### PR TITLE
Feature/dsbd 92

### PR DESCRIPTION
**Pull request by n3op2**
### What

A very basic story -> https://digicatapult.atlassian.net/browse/DSBD-92 for rendering Demo 2A desktop items.

### Changes
- new asset files
- removed some coded
- dekstop icons are within fixtures new

### 
Screenshot 2022-09-20 at 11 31 17
Screengrabs

### CODE REVIEW

File: src/components/demos/write/Demo.js [hunk 1]
```diff
@@ -83,7 +106,7 @@ export default function WriteDemo(props) {
       <Header {...props} showClose={true} />
       <Wrapper {...theme.wrapper}>
         {successfulLogin(apiOutput) ? (
-          <SecretDesktop {...theme.font} />
+          renderDesktopIcons(props.secretDesktop)
```
Comment (mattdean-digicatapult) [inline src/components/demos/write/Demo.js:109]: Should be rendered using `jsx` notation as `renderDesktopIcons` is essentially a functional `React` component

**Review by mattdean-digicatapult — CHANGES_REQUESTED** [review]
Very close, just one refactor/renaming I'd suggest

File: src/components/demos/write/Demo.js (hunk 1 above)
Comment (n3op2) [reply src/components/demos/write/Demo.js]: It has been updated.

File: src/components/demos/write/Demo.js [hunk 2]
```diff
@@ -22,10 +21,34 @@ const Wrapper = styled.div`
...
+            style={{ cursor: 'pointer' }}
+            width={'60px'}
+            height={'60px'}
+          />
+          <H2>{icon.name}</H2>
+        </IconWrapper>
+      ))}
+    </Col>
+  )
+}
```
[the remark this answers is not in the document]
Comment (n3op2) [reply src/components/demos/write/Demo.js]: Both comments were related, updated to render JSX instead

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**